ArcticaGreeter/ArcticaGreeter_RemoteLogon-HowTo.md

```diff
@@ -7,243 +7,315 @@ The following document, documents how to install Ubuntu with Arctica Greeter. Ma
+
+```
+# apt-get upgrade
+```
+
+This will download and Install the updates, this might take some time to complete. **NOTE:** it will ask you at the end if you want to continue, for this type **"y"** and press enter.
 
-	- **NOTE:** it will ask you at the end if you want to continue, for this type **"`y`"** and press enter.
 
-**NOTE: you can distinguish between Admin and User in the Terminal. A user has a `$-sign` when you are logged in a user, and `#-sign` if you are**
 
 ## Stage three 
 
 While you are still logged in as Admin/Superuser/ROOT you will need to add the repository.
 
 Type following:
-
-- **`nano /etc/apt/sources.list`**
+```
+nano /etc/apt/sources.list
+```
 
 Head to the bottom and press enter to insert new line and type following:
 
--	**`deb http://packages.arctica-project.org/ubuntu-nightly xenial main`**
+```
+deb http://packages.arctica-project.org/ubuntu-nightly xenial main
+```
 
-Now write out the repository by pressing **"`CTRL+O`"**, it will ask you to give the **"`File Name to Write`"** press enter to that.
+Now write out the repository by pressing **"CTRL+O"**, it will ask you to give the **File Name to Write** press enter to that.
 
-After you have saved it, press **"`CTRL+X`"** to exit the repository.
+After you have saved it, press **"CTRL+X"** to exit the repository.
 
 You should now be back in the Terminal, type the following command to fetch the key from a server:
 
--	**`apt-key adv --keyserver keyserver.ubuntu.com --recv-keys A8680F5598DE3101`**
+```
+apt-key adv --keyserver keyserver.ubuntu.com --recv-keys A8680F5598DE3101
+```
 
-It will tell you that a **"`RSA: 1 is imported`"** if it succeeded to import the key.
+It will tell you that a **RSA: 1 is imported** if it succeeded to import the key.
 
-Next you need to install git, type following command, it will ask you if you want to continue type **"`y`"** and press enter:
+Next you need to install git, type following command, it will ask you if you want to continue type **"y"** and press enter:
 
--	**`apt-get install git`**
+```
+apt-get install git
+```
 
 When the installation of Git is done, you will need to create a folder in order to clone git so they are collected in one folder. Therefore you need to type following command so you can create a folder:
 
--	**`mkdir arctica-greeter`**
+```
+mkdir arctica-greeter
+```
 
 Now open the folder that you just created by using following command:
 
--	**`cd arctica-greeter`**
+```
+cd arctica-greeter
+```
 
 Now you are able to clone the 4 repositories from ArcticaProject, so type following lines one by one:
 
--	**`git clone https://github.com/ArcticaProject/arctica-greeter/`**
--	**`git clone https://github.com/ArcticaProject/libpam-x2go/`**
--	**`git clone https://github.com/ArcticaProject/lightdm-remote-session-x2go/`**
--	**`git clone https://github.com/ArcticaProject/remote-logon-service/`**
+```
+git clone https://github.com/ArcticaProject/arctica-greeter/
```

Comment:
Prepent "$" or "#", depending on wether this is done as root or user.

No empty lines inbetween the commands, please.